ext/generate/generate_const.rb

```diff
@@ -115,7 +115,7 @@ module WMQ
       str << "\n# #{item[0]}\n"
       str << GenerateConst.rb_const("#{path}/#{item[1]}", item[2], item[3])
     end
-    GenerateConst.extract_const(path+'cmqc.h', 'MQ', /Queue\s+Attributes/, /General\s+Constants/).each do |item|
+    GenerateConst.extract_const(path+'cmqc.h', 'MQ', /Queue\s+Attributes/, /typedef\s+struct/).each do |item|
```

Comment:
Can you double check if this change is essential. It is intended for the C header file parser to know which constants to extract?
However, with each new MQ version this can change. The upside is that all constants regardless of MQ version all constants are defined.